chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +209,30 @@ def usage
           self.owner
         end
 
+        def multipath?
+          self.class.multipath?(@device)
+        end
+
+        def multipath_path?
```

Comment:
This method name is a bit ambiguous.  How about `held_by_multipath?` or similar?